colprac/README.md

```diff
@@ -0,0 +1,104 @@
+*   PR authors should not approve their own PRs.
+*   PRs should pass CI tests before being merged.
+*   PRs by people __without__ merge rights must have approval from someone who has merge rights (who will usually then merge the PR).
+*   PRs by people __with__ merge rights must have approval from someone else, who may or may not have merge rights (and then may merge their own PR).
+*   PRs by people with merge rights should not be merged by people other than the author (just approved).
+
+#### Releases
+
+*   A release should be made as soon as possible after a bugfix PR is merged.
+*   Care and consideration should be given as to when to make a breaking release.
+*   The person who merged the PR should register the new release of the package.
+
+---
+
+# ColPrac: Further Guidance
+
+This page offers some further guidance on conventions that can be helpful when collaborating on projects.
+This is an expansion on the Collaborative Practices, with more details and extra guidance.
+Anything detailed here should be considered less important than the main Collaborative Practices.
+
+## Guidance on contributing PRs
+
+*   You should usually open an issue about a bug or possible improvement before opening a PR with a solution.
+*   PRs should do a single thing, so that they are easier to review.
+    *   For example, fix one bug, or update compatibility, rather than fixing a bunch of bugs, and updating compatibility, and adding a new feature.
+*   PRs should add tests which cover the new or fixed functionality.
+
+*   PRs with large improvements to style should not also change functionality.
+
+*   PRs introducing breaking changes should make this clear when opening the PR.
+*   You should not push commits with commented-out tests.
+    *   Commenting out tests while developing locally is okay, but committing a commented-out test increases the risk of it silently not being run when it should be.
+*   You should not squash down commits while review is still ongoing.
+
+    *   Squashing commits prevents the reviewer from seeing what commits have been added since the last review.
+*   You should help __review__ your PRs, even though you cannot __approve__ your own PRs.
+
+    *   For instance, start the review process by commenting on why certain bits of the code changed, or highlighting places where you would particularly like reviewer feedback.
+
+## Guidance on reviewing PRs
+
+*   Review comments should be phrased as questions, as it shows you are open to new ideas.
+
+    *   For instance, “Why did you change this to X? Doesn’t that prevent Y?” rather than “You should not have changed this, it will prevent Y”.
+*   Small review suggestions, such as typo fixes, should make use of the `suggested change` feature.
+    *   This makes it easier and more likely for all the smaller changes to be made.
+*   Reviewers should continue acting as reviewers until the PR is merged.
+
+### Accidental breaking releases
```

Comment:
I guess this info is exclusive to repo maintainers, right?

Reply:
You mean, the Accidental breaking release?

Reply:
Answered as asked by Sam.

Reply:
Yes, the point is that general developers should not be the ones dealing with this type of issue.

Reply:
Added this information.